src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +11,11 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+
+        UserService userService = new UserService();
```

Comment:
What is the reason to create this object each time method login is called?

Reply:
How else I could call method findByEmail if I wouldnt create this object?